networks/bsc/bsc.md

```diff
@@ -25,7 +25,8 @@ gsn-relay-config.json:
   "workerMinBalance" : 10000000000000000,
   "workerTargetBalance" : 10000000000000000,
   "managerMinBalance": 10000000000000000,
-  "readyTimeout": 1200000
+  "readyTimeout": 1200000,
```

Comment:
Thanks for the PR! can you remove line 22 which is a duplicate of the (misconfigured) parameter?

Reply:
Done